components/Org/OrgDetail.js

```diff
@@ -1,93 +1,137 @@
+    display: block;
+    padding: 1rem 0;
+  }
+
+  Button {
+    background: #6549AA;
+    color: white;
+    width: 140px;
+    height: 50px;
+  }
 `
 
-const SocialButton = styled(Button)`
-  margin-top: 0.5rem;
-  font-size: 2rem !important;
+const OrgHeader = styled.h1`
+  font-size: 2rem;
+  letter-spacing: -1.1px;
+  font-weight: 700;
+  color: black;
+  padding-bottom: 1rem;
 `
 
 const ProfileContainer = styled.div`
-  margin: 0 auto;
-  position: relative;
-  @media screen and (min-width: 768px) and (max-width: 1280px) {
-    width: calc(100% - 4rem);
-    margin: initial;
-  }
+  display: grid;
+  grid-template-columns: 100%;
+  align-self: center;
+  justify-self: center;
 
-  @media screen and (max-width: 767px) {
-    width: calc(100% - 2rem);
-    margin: initial;
+  @media screen and (min-width: 767px) {
+    grid-template-columns: 25% 75%;
   }
 `
+
 const ProfileContentContainer = styled.div`
   width: 80rem;
-  margin: 4rem auto;
+  margin: 4rem auto 0;
   overflow: hidden;
-
-  @media screen and (min-width: 768px) and (max-width: 1280px) {
-    width: calc(100% - 4rem);
-    margin: initial;
-  }
-
-  @media screen and (max-width: 767px) {
-    width: calc(100% - 2rem);
-    margin: initial;
-  }
+  width: 100%;
 `
 
 const ProfileImage = styled.img`
-  margin: 0 calc(50% - 100px);
   width: 200px;
   object-fit: cover;
+  align-self: center;
+  justify-self: center;
 `
 
 const TabContainer = styled.div`
   text-align: left;
-  width: 50rem;
-  margin: 4rem auto;
-  overflow: hidden;
-  @media screen and (min-width: 768px) and (max-width: 1280px) {
-    width: calc(100% - 4rem);
-    margin: initial;
+  display: grid;
+  grid-template-columns: 100%;
+  margin-top: 3rem;
+  margin-bottom: 3rem;
+  width: 100%;
+
+  @media screen and (min-width: 767px) {
+    grid-template-columns: 20% 80%;
   }
+`
+
+const ContactList = styled.ul`
+  display: grid;
+  grid-template-columns: 45% 45%;
+  grid-column-gap: 5%;
+  padding: 0;
 
-  @media screen and (max-width: 767px) {
-    width: calc(100% - 2rem);
-    margin: initial;
+  @media screen and (min-width: 767px) {
+    grid-template-columns: 30% 30% 30%;
+    grid-column-gap: 3%;
+  }
+
+  li {
+    list-style: none;
+    color: black;
+    font-size: 1.2rem;
+    font-weight: 700;
+    margin-bottom: 2rem;
+
+    a {
+      font-size: 1rem;
+      overflow-wrap: break-word;
+
+      @media screen and (min-width: 767px) {
+        font-size: 1rem;;
+      }
+
+      @media screen and (min-width: 988px) {
+        font-size: 2rem;;
+      }
+    }
   }
 `
 
 const orgTab = (
-  <span style={{ fontSize: '1.2rem', fontWeight: 'bold' }}>
-    <Icon type='info-circle' />
+  <span style={{ fontSize: '1rem', fontWeight: 'bold', color: '#6549AA' }}>
     <FormattedMessage id='orgAbout' />
   </span>
 )
 
 const orgMemberTab = (
-  <span style={{ fontSize: '1.2rem', fontWeight: 'bold' }}>
-    <Icon type='team' />
+  <span style={{ fontSize: '1rem', fontWeight: 'bold', color: '#6549AA' }}>
```

Comment:
Same here - two fragments using the same style - if we want a bold purple label then we should put it in the theme or base css class so that it can be used consistent.y

Reply:
Made a component called OrgTabs and moved to Vtheme.